Content.Shared/Stunnable/SharedStunSystem.cs

```diff
@@ -256,12 +256,12 @@ private void OnAttackAttempt(EntityUid uid, StunnedComponent stunned, AttackAtte
             args.Cancel();
         }
 
-        private void OnEquipAttempt(EntityUid uid, StunnedComponent stunned, EquipAttemptEvent args)
+        private void OnEquipAttempt(EntityUid uid, StunnedComponent stunned, IsEquippingAttemptEvent args)
         {
             args.Cancel();
         }
 
-        private void OnUnequipAttempt(EntityUid uid, StunnedComponent stunned, UnequipAttemptEvent args)
+        private void OnUnequipAttempt(EntityUid uid, StunnedComponent stunned, IsUnequippingAttemptEvent args)
         {
             args.Cancel();
         }
```

Comment:
I'm not to sure what the best way to fix this is, but currently this also prevents you from stripping someone if they are stunned.

Maybe stripping should just bypass the (un)equipAttemptEvents?